src/node/waveshaper.rs

```diff
@@ -349,18 +350,19 @@ impl WaveShaperRenderer {
     }
 
     #[inline]
-    fn no_process(&self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
+    fn pass_through(&self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
         for (i_data, o_data) in input.channels().iter().zip(output.channels_mut()) {
-            for (&i, o) in i_data.iter().zip(o_data.iter_mut()) {
-                *o = i;
-            }
+            o_data.copy_from_slice(&i_data[..]);
         }
     }
 
     #[inline]
-    fn process_none(&self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
+    fn process_1x(&self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
+        // this does nothing...
```

Comment:
What do you mean here? We're applying the shape on the input to produce output, no?

Reply:
Sorry, just a comment for myself I forgot to remove